src/index.js

```diff
@@ -1,9 +1,11 @@
-/* eslint-disable no-unused-vars */
+/* eslint-disable */ /*importing files from the modules*/
 import _ from 'lodash';
```

Comment:
- [ ] Please remove any eslint disablement lines inside your code. Linters is a great tool that helps you write a clean and more readable code so disabling it is not a good practice.